en/guides/using-open-source-software.md

```diff
@@ -178,6 +178,8 @@ The development team that made the changes will be responsible for maintaining t
 To make changes to open source software, engage with the community and submit changes upstream to ensure that they are supported by future updates.
 See [Guide for Contributing to Open Source Software][guide-contribute-oss].
 
+Note: this is easily confused with the process of forking projects on GitHub which is critical in submtting changes (Pull Requests) back to the original project. 
```

Comment:
```suggestion
Note: this is easily confused with the process of forking projects on GitHub which is critical in submitting changes (Pull Requests) back to the original project. 
```

Reply:
I agree, this can be misleading but it's due to the term used by GitHub. I believe the term "fork" on GitHub is technically a remote cloning...

Since the guide was to be agnostic of the tool/platform, we may need some disambiguation and a clearer intended outcome.

Reply:
Yes. Maybe there could just be an "*" with a side note about use of remote cloning (forking). :)

I just wanted propose something to avoid confusion.

Reply:
Totally makes sense! I'll readjust the wording with the intent of not maintaining a duplicate project, to favour contributing changes back as much as possible since it's the right thing to do collectively but also because it means that those changes are thus maintained by the community instead of a handful of employees within your team who may move on at some point.